src/as2fm/as2fm_common/ecmascript_interpretation.py

```diff
@@ -22,7 +22,7 @@
 
 import js2py
 
-from as2fm_common.common import ValidTypes
```

Comment:
I wonder if we should rename `as2fm_common` to `common` and expose the parts needed by other packages by using the `__init__.py` file